Core/Lib/Extensions/JsonElementExtensions.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+
+namespace Lens.Core.Lib.Extensions
+{
+    public static class JsonElementExtensions
```

Comment:
There is a lot of 'Spaghetti code' in these extension methods. This code is different than what we have in the core framework. It would be hard to maintain that kind of code and use on multiple projects.

Reply:
I guess it has specific uses, not too often when you deal with Json at this level.